src/ui.rs

```diff
@@ -133,57 +133,56 @@ fn draw_messages<B: Backend>(f: &mut Frame<B>, app: &App, area: Rect) {
         .unwrap_or(0);
 
     let width = area.width - 2; // without borders
-    let max_lines = area.height;
 
     let time_style = Style::default().fg(Color::Yellow);
-    let messages = messages
-        .iter()
-        .rev()
-        // we can't show more messages atm and don't have messages navigation
-        .take(max_lines as usize)
-        .map(|msg| {
-            let arrived_at = msg.arrived_at.with_timezone(&chrono::Local);
-
-            let time = Span::styled(
-                format!("{:02}:{:02} ", arrived_at.hour(), arrived_at.minute()),
-                time_style,
-            );
-            let from = displayed_name(&msg.from, app.config.first_name_only);
-            let from = Span::styled(
-                textwrap::indent(&from, &" ".repeat(max_username_width - from.width())),
-                Style::default().fg(user_color(&msg.from)),
-            );
-            let delimeter = Span::from(": ");
-
-            let displayed_message = displayed_message(&msg);
-
-            let prefix_width = (time.width() + from.width() + delimeter.width()) as u16;
-            let indent = " ".repeat(prefix_width.into());
-
-            let wrap_opts = textwrap::Options::new(width.into())
-                .initial_indent(&indent)
-                .subsequent_indent(&indent);
-            let lines = textwrap::wrap(displayed_message.as_str(), wrap_opts);
-
-            let spans: Vec<Spans> = lines
-                .into_iter()
-                .enumerate()
-                .map(|(idx, line)| {
-                    let res = if idx == 0 {
-                        vec![
-                            time.clone(),
-                            from.clone(),
-                            delimeter.clone(),
-                            Span::from(line.strip_prefix(&indent).unwrap().to_string()),
-                        ]
-                    } else {
-                        vec![Span::from(line.to_string())]
-                    };
-                    Spans::from(res)
-                })
-                .collect();
-            spans
-        });
+    let messages = messages.iter().rev().map(|msg| {
+        let arrived_at = msg.arrived_at.with_timezone(&chrono::Local);
+
+        let time = Span::styled(
+            format!(
+                "{:02} {:02}:{:02} ",
```

Comment:
I guess at some point we should make it configurable. But this is a better default in any case.